src/ec/system76/ec/Makefile.inc

```diff
@@ -6,4 +6,8 @@ all-y += system76_ec.c
 all-y += buttons.c
 smm-$(CONFIG_DEBUG_SMI) += system76_ec.c
 
+cbfs-files-$(CONFIG_EC_SYSTEM76_EC_UPDATE) += ec.rom
+ec.rom-file :=$(call strip_quotes,$(CONFIG_EC_SYSTEM76_EC_UPDATE_FILE))
+ec.rom-type := raw
```

Comment:
If this happens in ramstage we can compress it right? Waste of 128KB of space, especially that we do not fully occupy the EC flash space. Free space compresses greatly

Reply:
I had some issues with decompression due to cbfs cache not being available, i'll try again

Reply:
needed to bump the CBFS cache size. Done in 0874878375db60aaaff43640c14d8f46bc9c41ca

Reply:
Hmm. Maybe it should use cbfs_load instead of cbfs_map. That way we will be loading the EC FW to memory, instead of caching it from flash. Also it would be much more safe with CBFS_VERIFICATION.

Reply:
implemented in https://github.com/Dasharo/coreboot/pull/481/commits/5ca878849e9d01975f570c478629bc3689a438d5